Using TensorFlow backend.
WARNING:tensorflow:From /c1/apps/anaconda/4.3.1/lib/python3.5/site-packages/tensorflow/contrib/learn/python/learn/datasets/base.py:198: retry (from tensorflow.contrib.learn.python.learn.datasets.base) is deprecated and will be removed in a future version.
Instructions for updating:
Use the retry module or similar alternatives.
WARNING:tensorflow:From /c1/apps/anaconda/4.3.1/lib/python3.5/site-packages/keras/backend/tensorflow_backend.py:1150: calling reduce_sum (from tensorflow.python.ops.math_ops) with keep_dims is deprecated and will be removed in a future version.
Instructions for updating:
keep_dims is deprecated, use keepdims instead
WARNING:tensorflow:From /c1/apps/anaconda/4.3.1/lib/python3.5/site-packages/keras/backend/tensorflow_backend.py:1114: calling reduce_max (from tensorflow.python.ops.math_ops) with keep_dims is deprecated and will be removed in a future version.
Instructions for updating:
keep_dims is deprecated, use keepdims instead
/home/straynwang/Job_matches/./experiments/models/LSTM_15.py:25: UserWarning: The `merge` function is deprecated and will be removed after 08/2017. Use instead layers from `keras.layers.merge`, e.g. `add`, `concatenate`, etc.
  output_attention_mul = merge([inputs, a_probs], name='attention_mul', mode='mul')
/c1/apps/anaconda/4.3.1/lib/python3.5/site-packages/keras/legacy/layers.py:460: UserWarning: The `Merge` layer is deprecated and will be removed after 08/2017. Use instead layers from `keras.layers.merge`, e.g. `add`, `concatenate`, etc.
  name=name)
/home/straynwang/Job_matches/./experiments/models/LSTM_15.py:63: UserWarning: Update your `Model` call to the Keras 2 API: `Model(outputs=Tensor("de..., inputs=[<tf.Tenso...)`
  model = Model(input=[inputs], output=x)
WARNING:tensorflow:From /c1/apps/anaconda/4.3.1/lib/python3.5/site-packages/keras/backend/tensorflow_backend.py:2852: calling l2_normalize (from tensorflow.python.ops.nn_impl) with dim is deprecated and will be removed in a future version.
Instructions for updating:
dim is deprecated, use axis instead
WARNING:tensorflow:From /c1/apps/anaconda/4.3.1/lib/python3.5/site-packages/keras/backend/tensorflow_backend.py:1257: calling reduce_mean (from tensorflow.python.ops.math_ops) with keep_dims is deprecated and will be removed in a future version.
Instructions for updating:
keep_dims is deprecated, use keepdims instead
Traceback (most recent call last):
  File "./main.py", line 150, in <module>
    exp = start_exp(exp)
  File "./main.py", line 107, in start_exp
    log_path=path_training_log)
  File "/home/straynwang/Job_matches/./lib/utils.py", line 46, in train_model
    shuffle=shuffle, validation_split=validation_split)
  File "/c1/apps/anaconda/4.3.1/lib/python3.5/site-packages/keras/engine/training.py", line 1429, in fit
    batch_size=batch_size)
  File "/c1/apps/anaconda/4.3.1/lib/python3.5/site-packages/keras/engine/training.py", line 1305, in _standardize_user_data
    exception_prefix='input')
  File "/c1/apps/anaconda/4.3.1/lib/python3.5/site-packages/keras/engine/training.py", line 139, in _standardize_input_data
    str(array.shape))
ValueError: Error when checking input: expected input_1 to have shape (None, 200, 1, 1) but got array with shape (13841, 1, 1, 200)
